### PR TITLE
#168357746 Fix trip id validation error

### PR DESCRIPTION
**Pull request by Pomile**
#### Why
This is a proposed fix for Reject request(by requester Manager) [#167891587](https://www.pivotaltracker.com/story/show/167891587)
#### Description of task to be done
- [x] Write a test
- [x] Add a function that validate UUID
- [x] Add validation for tripId in the request parameter

##### Error
tripStatus1
##### Solved
tripStatus

### CODE REVIEW

**Review by max-wel — COMMENTED** [review]
What if you use ```param('tripId').isUUID(4)``` instead of creating a custom validator and helper.

File: src/utils/uuidValidator.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (thatkazuk1) [inline src/utils/uuidValidator.js]: You can simply use express-validator `isUUID(4)` here. There won't be any need for this helper

Comment (Pomile) [reply src/utils/uuidValidator.js]: Noted

**Comment by Pomile** [comment]
> What if you use `param('tripId').isUUID(4)` instead of creating a custom validator and helper.

Noted

**Comment by Pomile** [comment]
> What if you use `param('tripId').isUUID(4)` instead of creating a custom validator and helper.

Done

**Review by max-wel — APPROVED** [review]
Good job

**Review by daniellamarr — APPROVED** [review]

**Pull request merged**